modules/gitops-disable-dex.adoc

```diff
@@ -0,0 +1,21 @@
+// Module is included in the following assemblies:
+//
+// * configuring-sso-for-argo-cd-on-openshift
+
+[id="disable-dex_{context}"]
```

Comment:
```suggestion
[id="gitops-disable-dex_{context}"]
```
The `id` value needs to be the same as the filename per the OCP docs guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata